lib/ui/imagedecoratedstructs.flow

```diff
@@ -13,14 +13,15 @@ export {
 	ImageDecoration ::= PreZoomDecoration, ImageCrop, ImageAdaptiveCrop, ImageThumbnailCrop,
 		PostZoomDecoration, SizeIndependentDecoration, ImageAdaptiveCropTweaks;
 
-		PreOrPostZoomDecoration ::= ImageLabel, ImageCircle, ImageArrow, ImageRectangle, ImageTextBox;
+		PreOrPostZoomDecoration ::= ImageLabel, ImageCircle, ImageArrow, ImageRectangle, ImageTextBox, ImageTextColumn;
 
 		PreZoomDecoration ::= PreOrPostZoomDecoration, ImageDimmer; //Are grouped with full size image and then zoomed
 			ImageArrow   : (x : double, y : double, rotate : double, length : double, color : int, thickness : double, label : Form);
 			ImageCircle  : (x : double, y : double, radius : double, style : [GraphicsStyle], inverted : bool);
 			ImageLabel   : (x : double, y : double, label : Form); //Text is replaced with Form  as "Empty" can be found in serialized data
 			ImageRectangle : (x : double, y : double, width : double, height : double, style : [GraphicsStyle], inverted : bool);
 			ImageTextBox : (label : Text, hAlign : CommonAlignment, vAlign : VerticalAlignment, rectangle : ImageRectangle);
+			ImageTextColumn(label : Text, x : double, y : double, width : double, hAlign : CommonAlignment, vAlign : VerticalAlignment, style : [GraphicsStyle]);
```

Comment:
Also here, double is more flexible, and convenient in part where you adjust your text vertically. Single multiplication instead of cumbersome switch.